corehq/apps/saved_reports/management/commands/fix_saved_reports_filters.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from dimagi.utils.couch.cache import cache_core
+
+from corehq.apps.saved_reports.models import ReportConfig
+from corehq.apps.userreports.reports.filters.values import CHOICE_DELIMITER
+
+
+class Command(BaseCommand):
+    """
+        Migrated ReportConfig filters with multiple values from CHOICE_DELIMITER-delimited strings
+        to lists. This command can be deleted once that migration is complete.
+    """
+    def handle(self, *args, **options):
+        key = ["name slug"]
+        results = cache_core.cached_view(
+            ReportConfig.get_db(),
+            "reportconfig/configs_by_domain",
+            reduce=False,
+            include_docs=True,
+            startkey=key,
+            endkey=key + [{}]
+        )
+        count = 0
+        for result in results:
+            dirty = False
+            doc = result['doc']
+            config = ReportConfig.get(doc['_id'])
```

Comment:
This goes back to the DB to get the doc, no? Why can't you just call `.wrap` since you're already including docs? Or maybe better yet, use `iter_docs`?

Reply:
Switched to `wrap` in 6a8e7db400e